markdown.go

```diff
@@ -1082,6 +1089,21 @@ func expandTabs(out *bytes.Buffer, line []byte, tabSize int) {
 	}
 }
 
+// Extract indent finds the appropriate indentation for the next block
+// where line can be inside of a list.
+func extractIndent(line []byte, tabSize int) []byte {
```

Comment:
Not a refactor, it was a necessary function to implement code including during the first pass. This extracts the indentation for a code include line and uses it to indent all the lines in code-block.